external/js/cothority/src/log.ts

```diff
@@ -75,7 +73,7 @@ export class Logger {
             // @ts-ignore
             return (file).padEnd(20);
         } catch (e) {
-            return this.printCaller(new Error("Couldn't get stack - " + e), i + 2);
+            return this.out("Couldn't get stack - " + e.toString(), (i + 2).toString());
```

Comment:
Where does this magic value of 2 come from? 
There seems to be an unnecessary extra space.

Reply:
It's because the stack includes the `printCaller` and `{lvl*,print,...}` method.